app-android/src/main/java/com/ustadmobile/port/android/view/ClazzEditFragment.kt

```diff
@@ -284,4 +317,393 @@ class ClazzEditFragment() : UstadEditFragment<ClazzWithHolidayCalendarAndSchoolA
+                text = { Text(stringResource(id = R.string.add_block)) },
+            )
+        }
+
+        items (
+            items = uiState.courseBlockList
+        ) { courseBlock ->
+
+            val image = if(courseBlock.cbType == CourseBlock.BLOCK_CONTENT_TYPE)
+                courseBlock.entry?.contentTypeFlag
+            else
+                courseBlock.cbType
+
+            val courseBlockEditAlpha: Float = if (courseBlock.cbHidden) 0.5F else 1F
+            val startPadding = (courseBlock.cbIndentLevel * 8).dp
+
+            ListItem(
+                modifier = Modifier
+                    .padding(start = startPadding)
+                    .alpha(courseBlockEditAlpha),
+                icon = {
+                    Row{
+                        Icon(
+                            Icons.Filled.Menu,
+                            contentDescription = null
+                        )
+                        Icon(
+                            painterResource(id = CONTENT_ENTRY_TYPE_ICON_MAP[image] ?: R.drawable.text_doc_24px),
+                            contentDescription = null
+                        )
+                    }
+                },
+
+                text = { Text(courseBlock.cbTitle ?: "") },
+                trailing = {
+                    PopUpMenu(
+                        enabled = uiState.fieldsEnabled,
+                        entity = courseBlock,
+                        onClickHideBlockPopupMenu = onClickHideBlockPopupMenu,
+                        onClickIndentBlockPopupMenu = onClickIndentBlockPopupMenu,
+                        onClickUnIndentBlockPopupMenu = onClickUnIndentBlockPopupMenu,
+                        onClickDeleteBlockPopupMenu = onClickDeleteBlockPopupMenu,
+                    )
+                }
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(20.dp))
+        }
+
+        item {
+            Text(text = stringResource(id = R.string.schedule))
+        }
+
+        item {
+            ListItem(
+                modifier = Modifier.clickable {
+                    onClickAddSchedule()
+                },
+                text = { Text(stringResource(id = R.string.add_a_schedule)) },
+                icon = {
+                    Icon(
+                        imageVector = Icons.Filled.Add,
+                        contentDescription = "",
+                    )
+                }
+            )
+        }
+
+        items(
+            uiState.clazzSchedules
+        ){ schedule ->
+
+            val fromTimeFormatted = rememberFormattedTime(timeInMs = schedule.sceduleStartTime.toInt())
+            val toTimeFormatted = rememberFormattedTime(timeInMs = schedule.scheduleEndTime.toInt())
+            val text = "${messageIdResource(id = schedule.scheduleFrequency)} " +
+                    " ${messageIdResource(schedule.scheduleDay)} " +
+                    " $fromTimeFormatted - $toTimeFormatted "
+
+            ListItem(
```

Comment:
Please add an invisible icon (e.g. alpha = 0)  to the schedule ListItem so that the alignment matches with the add button.